cli/ingress-controller/main.go

```diff
@@ -351,18 +344,41 @@ func main() {
 		UpdateCh: updateChannel,
 	}
 
+	var preferredIngressAPIs []utils.IngressAPI
+	if cliConfig.AllowIngressNetworkingV1 {
```

Comment:
The commits are arranged wrong. This struct member is introduced in the next commit. That threw me off. I've a question in the next commit about the configuration option itself.
Just noting here, no changes required.

Reply:
Sorry about throwing you off.
I can squash these two commits if you wish.